app/Filament/Resources/UserResource/Pages/ListUsers.php

```diff
@@ -239,6 +243,29 @@ function (Builder $query, array $data) {
                     ->multiple()
                     ->searchable()
                     ->preload(),
+
+                Filter::make('created_after')
+                    ->form([
+                        DateTimePicker::make('created_at')
+                            ->label('Created After')
+                            ->format('m/d/Y H:i:s')
+                            ->displayFormat('m/d/Y H:i:s')
+                            ->native(false),
+                    ])
+                    ->query(function (Builder $query, array $data): Builder {
+                        return $query
+                            ->when(
+                                $data['created_at'],
+                                fn (Builder $query, $date): Builder => $query->where('created_at', '>=', Carbon::parse($date, app(DisplaySettings::class)->getTimezone() ?? config('app.timezone'))->setTimezone('UTC')),
```

Comment:
If you look at the `getTimezone` method you will see that it defaults to that config value.

```suggestion
                                fn (Builder $query, $date): Builder => $query->where('created_at', '>=', Carbon::parse($date, app(DisplaySettings::class)->getTimezone())->setTimezone('UTC')),
```

Reply:
Done